frontend/src/views/ensemble/index.tsx

```diff
@@ -22,7 +41,25 @@ const EnsembleView: FC = () => {
       >
         <Item>
           <h3>Instrument Settings</h3>
-          <p>placeholder</p>
+          <Box sx={{ width: "50%", marginTop: "5px", marginBottom: "20px" }}>
+            <FormControl fullWidth>
+              <InputLabel id="demo-simple-select-label">Instrument</InputLabel>
```

Comment:
You should either:
1. Change the id to make more sense, and add a `labelId` property to `Select`. or
2. Delete the unused id property from InputLabel